freqtrade/freqai/freqai_interface.py

```diff
@@ -672,7 +705,8 @@ def fit_live_predictions(self, dk: FreqaiDataKitchen, pair: str) -> None:
         for label in full_labels:
             if self.dd.historic_predictions[dk.pair][label].dtype == object:
                 continue
-            f = spy.stats.norm.fit(self.dd.historic_predictions[dk.pair][label].tail(num_candles))
+            f = spy.stats.norm.fit(
+                self.dd.historic_predictions[dk.pair][label].fillna(0).tail(num_candles))
```

Comment:
This is dangerous - but maybe there is a good reason to do it? I cant imagine we will have NaNs very often in the historic_predictions dataframe - so is this only for startup? 

Otherwise, this modifies the statistics.

Reply:
Fixed.